.ci/setup_env_centos.sh

```diff
@@ -88,3 +89,11 @@ chronic sudo -E yum install -y procenv
 
 echo "Install haveged"
 chronic sudo -E yum install -y haveged
+
+if [ "$KATA_KSM_THROTTLER" == "yes" ]; then
+	package="kata-ksm-throttler"
+	echo "Install ${package}"
+	chronic sudo -E yum install ${package}
+	echo "Start ${package}"
+	sudo systemctl start ${package}
```

Comment:
More thinking aloud: I think that by default the service is both disabled and not started. I guess we don't need to run multi-boot tests so enabling isn't strictly necessary here.